src/unix/thread.c

```diff
@@ -37,6 +38,12 @@
 #include <sys/sem.h>
 #endif
 
+#ifdef UV_BSD_H
+#include <pthread_np.h>
+typedef cpuset_t cpu_set_t;
```

Comment:
`cpuset_t` is not defined on all BSDs, in particular OpenBSD does not seem to have it.

Reply:
Since https://github.com/libuv/libuv/commit/20cc969e5db3467bddaa35f48ba051a7ae3b3653#diff-993e2d1f75ebf418adfbe81df8050a95R48 was just merged into libuv master, this should now just copy what is done there